explainer.html

```diff
@@ -236,42 +259,52 @@ <h2>Use Cases and Requirements</h2>
       <dt id="detectingChanges">Detecting changes in Datasets</dt>
       <dd>
         When processing RDF Datasets over a period of time, determining if information has changed is helpful. For example, knowing if information has changed helps with data cache invalidation, detecting if expected data has been tampered with or modified, or when debugging unexpected changes in source RDF Datasets.
-        <br>Requirement: An RDF Dataset Canonicalization algorithm.
+        <br>Requirement: An RDF Dataset Canonicalization and Hash algorithms.
```

Comment:
```suggestion
        <br>Requirement: RDF Dataset Canonicalization and Hash algorithms.
```
Since "algorithms" is plural, "An" looks incorrect to me (strange, anyway).